core/src/retries.rs

```diff
@@ -37,83 +40,165 @@ pub fn http_request_exponential_backoff() -> ExponentialBackoff {
     }
 }
 
-/// An [`ExponentialBackoff`] with parameters tuned for tests where we don't want to be retrying
-/// for 10 minutes.
-#[cfg(feature = "test-util")]
-#[cfg_attr(docsrs, doc(cfg(feature = "test-util")))]
-pub fn test_http_request_exponential_backoff() -> ExponentialBackoff {
-    ExponentialBackoff {
-        initial_interval: Duration::from_nanos(1),
-        max_interval: Duration::from_nanos(30),
-        multiplier: 2.0,
-        max_elapsed_time: Some(Duration::from_millis(100)),
-        ..Default::default()
+/// HttpResponse represents an HTTP response. It will typically be returned from
+/// [`retry_http_request`].
+#[derive(Debug)]
```

Comment:
Any reason not to derive `Clone`? I feel like I never regret deriving `Clone` if it doesn't require any extra work.